react/react1/week1/todolist/src/components/BlackBorder.js

```diff
@@ -0,0 +1,5 @@
+function BlackBorder({ children }) {
```

Comment:
I would not do this as a component, simply just use a div and add the class around the place you want to use this class.

Reply:
I think it was required in a task to use children here

Reply:
Yeah this is my mistake, I did not realize it was a requirement, sorry.